src/Symfony/Component/DependencyInjection/Alias.php

```diff
@@ -102,23 +100,62 @@ public function setDeprecated(bool $status = true, string $template = null)
             if (false === strpos($template, '%alias_id%')) {
                 throw new InvalidArgumentException('The deprecation template must contain the "%alias_id%" placeholder.');
             }
+        }
+
+        $this->deprecation['message'] = $template ?: self::$defaultDeprecationTemplate;
```

Comment:
the `setDeprecated()` method should be deprecated also

Reply:
Done